lighthouse-core/report/templates/report-template.html

```diff
@@ -61,15 +61,19 @@
             <h1 class="menu__header-title">Lighthouse</h1>
             <div class="menu__header-version">Version: {{lighthouseVersion}}</div>
           </div>
-          <ul class="menu__nav">
-            {{#each aggregations}}
-            <li class="menu__nav-item">
-              <a class="menu__link" href="#{{nameToLink this.name}}">
-                {{ this.name }}
-              </a>
-            </li>
-            {{/each}}
-          </ul>
+          {{#if_eq reportContext "perf-x" }}
```

Comment:
would it be possible to branch this template not on reportContext, but on the data provided to the template?

so we have a partial for this leftnav thing... we have `_relatedReports` or w/e, then we display all those, otherwise, simple `menu__nav-item`s.